Gemfile

```diff
@@ -4,5 +4,5 @@ source 'https://rubygems.org' do
   gem 'fog'
   gem 'inspec'
   gem 'net-ssh'
-  gem 'optparse'
+  gem 'rubysl-optparse'
```

Comment:
Why this change?

Reply:
They changed the name recently. `optparse` no longer exists as a package.